postprocessing/resstockpostproc/resources/publication/sdr_column_definitions.csv

```diff
@@ -346,6 +348,43 @@ Output,report_utility_bills.utility_rates_fixed_variable_fuel_oil_total_usd,$,ou
+Output,report_simulation_output.electric_panel_load_clothes_dryer_w,W,out.panel.load.clothes_dryer.w,out.panel.load.clothes_dryer.w,w,1,,,,,,,"Electric load of clothes dryer, for electric only"
+Output,report_simulation_output.electric_panel_load_cooling_w,W,out.panel.load.cooling.w,out.panel.load.cooling.w,w,1,,,,,,,"Electric load of space cooling. For the total load calculation, the max of heating or cooling is used, not both"
+Output,report_simulation_output.electric_panel_load_dishwasher_w,W,out.panel.load.dishwasher.w,out.panel.load.dishwasher.w,w,1,,,,,,,Electric load of dishwasher
+Output,report_simulation_output.electric_panel_load_electric_vehicle_charging_w,W,out.panel.load.ev_charging.w,out.panel.load.ev_charging.w,w,1,,,,,,,Electric load of electric vehicle charging equipment
+Output,report_simulation_output.electric_panel_load_heating_w,W,out.panel.load.heating.w,out.panel.load.heating.w,w,1,,,,,,,"Electric load of space heating. For the total load calculation, the max of heating or cooling is used, not both"
+Output,report_simulation_output.electric_panel_load_hot_water_w,W,out.panel.load.hot_water.w,out.panel.load.hot_water.w,w,1,,,,,,,"Electric load of water heater, for electric only"
+Output,report_simulation_output.electric_panel_load_mech_vent_w,W,out.panel.load.mech_vent.w,out.panel.load.mech_vent.w,w,1,,,,,,,Electric load of mechanical ventilation
+Output,report_simulation_output.electric_panel_load_other_w,W,out.panel.load.other.w,out.panel.load.other.w,w,1,,,,,,,"Other electric load, including but not limited to lighting/general receptacles at 3 W/sqft, refrigerator, microwave, and garbage disposal."
+Output,report_simulation_output.electric_panel_load_permanent_spa_heater_w,W,out.panel.load.permanent_spa_heat.w,out.panel.load.permanent_spa_heat.w,w,1,,,,,,,"Electric load of permanent spa heater, for electric only"
+Output,report_simulation_output.electric_panel_load_permanent_spa_pump_w,W,out.panel.load.permanent_spa_pump.w,out.panel.load.permanent_spa_pump.w,w,1,,,,,,,Electric load of permanent spa pump
+Output,report_simulation_output.electric_panel_load_pool_heater_w,W,out.panel.load.pool_heater.w,out.panel.load.pool_heater.w,w,1,,,,,,,"Electric load of pool heater, for electric only"
+Output,report_simulation_output.electric_panel_load_pool_pump_w,W,out.panel.load.pool_pump.w,out.panel.load.pool_pump.w,w,1,,,,,,,Electric load of pool pump
+Output,report_simulation_output.electric_panel_load_range_oven_w,W,out.panel.load.range_oven.w,out.panel.load.range_oven.w,w,1,,,,,,,"Electric load of range oven, for electric only"
+Output,report_simulation_output.electric_panel_load_well_pump_w,W,out.panel.load.well_pump.w,out.panel.load.well_pump.w,w,1,,,,,,,Electric load of well pump
+Input,report_simulation_output.electric_panel_breaker_spaces_total_count,count,in.electric_panel_breaker_space_total_count,in.electric_panel_breaker_space_total_count,count,1,,,,,,,Total breaker spaces on electric panel

```

Comment:
This is intentionally listed as an input.

Reply:
Sorry, I don't follow. What are you trying to mirror and why is report_simulation_output variable considered an Input in this case?

Reply:
Because the actual input sampled by electric_panel.rb is the breaker spaces headroom count (but we pipe it as an  output), which is then used to determine the total count in the baseline. In the upgrade however, we pass in the total count from the baseline, recalculate the occupied space based on the upgraded house to determine the new headroom count. This is too confusing to explain and does not matter to the users.

Instead, I want to preserve the basic structure of headroom (output) = total rated (input) - occupied (output). This is what the panel service load calculation follows.

For capacity, the size or service rating of the panel is the input sampled by electric_panel.rb instead. This is synonymous with the total space count (or the rated number of breaker slots).

Reply:
Thanks!